### DISCUSSION TITLE
I took it from your documents, why is it like this

### DISCUSSION THREAD

**Discussion by wosiyourfather** [AvaloniaUI/Avalonia#12794] [Q&A]
![1693965960649](https://github.com/AvaloniaUI/Avalonia/assets/53590491/897b017b-745a-4c4c-9283-9aa06537124e)

**Comment by wosiyourfather** [comment]
Incomplete display in sub options

**Comment by wosiyourfather** [comment]
???

**Comment by rabbitism** [comment]
There are so many things that can impact the layout. I cannot tell anything without seeing the actual code. Can you provide a sample? 
And when you are debugging, you can always press F12 to launch devtool and check the actual bounds of elements.

**Reply by rabbitism** [reply]
And you can join QQ group 397510870 if you wanna talk with Chinese developers.

**Reply by wosiyourfather** [reply]
Thank you, group. I have applied for WPF and I also have this problem. I have a solution
